resources/benchmark-runner.mjs

```diff
@@ -403,10 +403,6 @@ export class BenchmarkRunner {
         const prepareEndLabel = "runner-prepare-end";
 
         performance.mark(prepareStartLabel);
-        this._removeFrame();
```

Comment:
do you know why this was called before all other preparation?

I did some archeology it seemed to be here from the very first version of this file https://github.com/WebKit/Speedometer/commit/a1e842164bc1803be6afa658bbb1336ecc726b12#diff-e1b90c3d88d846fffc55176c8279f44d1195bd93798b1ebd868a9494a21e4897R215

I'd say it's fine to remove it, it doesn't seem useful to me, but let's be prepared to a possible regression here.

Reply:
might have been related to re-running in the same window back then, but we definitely don't allow that directly anymore.